app/mailers/report_mailer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class ReportMailer < ApplicationMailer
+  default template_path: 'reports/mailer'
+  layout false
```

Comment:
Didn't need this.

"app/views/layouts/mailer.text" -> "app/views/layouts/mailter.text.haml" fixed the issue with the text email not rendering correctly.

Though, granted, this template file is essentially empty.